src/components/enter-phone-number/tests/enter-phone-number-integration.test.ts

```diff
@@ -337,10 +338,11 @@ describe("Integration::enter phone number", () => {
         _csrf: token,
         phoneNumber: "07738394991",
       })
-      .expect(
-        "Location",
-        `${PATH_NAMES.SECURITY_CODE_WAIT}?actionType=${SecurityCodeErrorType.OtpBlocked}`
-      )
+      .expect((res) => {
```

Comment:
We should update the test description of tests where we change the expectation as probably they're incorrect now